extensions/blocks/podcast-player/edit.js

```diff
@@ -92,13 +132,7 @@ const PodcastPlayerEdit = ( {
 			>
 				<form onSubmit={ checkPodcastLink }>
 					{ noticeUI }
-					<TextControl
-						type="url"
-						placeholder={ __( 'Enter URL here…', 'jetpack' ) }
-						value={ editedUrl || '' }
-						onChange={ setEditedUrl }
-						className={ 'components-placeholder__input' }
-					/>
+					<FeedURLControl url={ editedUrl || '' } onUrlChange={ setEditedUrl } />
```

Comment:
```suggestion
					<FeedURLControl url={ editedUrl  } onUrlChange={ setEditedUrl } />
```

Could we set defaults within the component?

Reply:
Yeah, in fact, [it already does it](https://github.com/Automattic/jetpack/pull/14973/files#diff-ac95db14f00bcb6ae1960269ae9121cfR72).